samples/kotlin_compose/src/androidTest/java/io/customer/android/sample/kotlin_compose/MemoryLeakageInstrumentedTest.kt

```diff
@@ -83,6 +85,22 @@ class MemoryLeakageInstrumentedTest {
             build()
         }
 
+        CustomerIOBuilder(
```

Comment:
Is this change in scope? It seems unrelated to speeding up the CI. No need to reply, just a thought.

Reply:
Not speeding up, but I think it is to fix this test as some SDK calls in sample app now rely on new SDK initialization. And if we don't initialize new instance, the code may keep breaking and tests will fail.